src/Microsoft.Health.Fhir.Core/Configs/CoreFeatureConfiguration.cs

```diff
@@ -101,5 +101,13 @@ public class CoreFeatureConfiguration
         /// Gets or sets a value indicating whether the server supports the includes.
         /// </summary>
         public bool SupportsIncludes { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether geo-redundancy monitoring is enabled.
+        /// When enabled, the system will monitor geo-replication lag and status through
+        /// the GeoReplicationLagWatchdog. This feature is only applicable when using
+        /// Azure SQL Database with geo-replication configured.
+        /// </summary>
+        public bool EnableGeoRedundancy { get; set; }
```

Comment:
Can you update appsettings to include this parameter. Even if it is set to false I think it is better to explicitly define each config value in appsettings.